files/default/neutron-ha-tool.py

```diff
@@ -248,6 +248,14 @@ def run(args):
     return 1 if errors > 0 else 0
 
 
+class RandomAgentPicker(object):
+    def __init__(self, agents):
+        self.agents = agents
+
+    def pick(self):
```

Comment:
I suggested `choice` because of the python API. I guess that `pick` is ok, even to my eyes is too similar to pickle